packages/api/db/tables.sql

```diff
@@ -0,0 +1,28 @@
+-- User tags are associated to a user for the purpose of granting/restricting them
+-- in the application.
+-- Needed per https://www.postgresql.org/message-id/20141203213139.GA9855%40fetter.org
+CREATE TYPE user_tag_type AS ENUM
+(
+  'HasAccountRestriction',
+  'HasPsaAccess',
+  'HasSuperHotAccess',
+  'StorageLimitBytes'
+);
+
+-- A transaction in nftstorage.link perma cache.
+CREATE TABLE IF NOT EXISTS public.perma_cache
+(
+    id             BIGSERIAL PRIMARY KEY,
+    user_id        BIGINT                                                          NOT NULL,
+    url            TEXT                                                          NOT NULL,
```

Comment:
Only included `url` which always is `source_url` provided by user. In current implementation, we really only care for `normalized_url` for R2 key

Reply:
I would include `normalized_url` here incase we ever change our idea of what normal is.